portugues/revisa/tenta-consegue/ti-cruel/coco-na-praia.md

```diff
@@ -2,4 +2,5 @@ Voc
 cenda de cocos na praia. Por�m voc� est� tendo dificuldades para saber
 se est� vendendo cocos o suficiente. O que voc� faz?
 
-
+[Desenvolve um app para contagem de cocos e ratings dos melhores vendedores de coco](desenvolve-app/coco-app.md)
+[Come�a a comer seus pr�pris cocos e vai a fal�ncia](come/falencia.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.